podcast/2.11.md

```diff
@@ -0,0 +1,897 @@
+---
+title: 'Pioneering the developer advocate role, with Cassidy Williams, Director of Developer Experience at Netlify'
+publishDate: 2021-16-09T10:00-07:00
+tags: [podcast]
+slug: cassidy-williams
+published: true
+---
+
+<!-- START AUDIO -->
+
+<audio className="object-center" src="https://www.buzzsprout.com/1097978/9437422-han-wen-nienhuys-staff-software-engineer-at-google.mp3?download=true" controls preload="none"></audio>
+
+<!-- END AUDIO -->
+
+<!-- START GUESTS -->
+<span>
+Cassidy Williams, Beyang Liu
+</span>
+<!-- END GUESTS -->
+
+<!-- START SUMMARY -->
+
+How can you build a following, and a career, with memes? In this episode, Cassidy Williams, Director of Developer Experience at Netlify, joins Beyang Liu, co-founder and CTO of Sourcegraph, to discuss why we should consider communication a core skill instead of a soft skill, why you should be a developer advocate or a software engineer but not both, and why, when learning React, you should start with the fundamentals. Along the way, Cassidy shares stories about the job she held the longest (mascot for Iowa State), positive and negative experiences from the heyday of hackathons, and the time she nearly went blind from burnout.
+
+Click the audio player below to listen!
+
+<!-- END SUMMARY -->
+
+<!-- START SHOWNOTES -->
+
+Lilypond: http://lilypond.org/
+
+Why Google Stores Billions of Lines of Code in a Single Repository: https://cacm.acm.org/magazines/2016/7/204032-why-google-stores-billions-of-lines-of-code-in-a-single-repository/fulltext
+
+FUSE bindings for Go: https://github.com/hanwen/go-fuse
+
+Bazel: https://bazel.build/
+
+Gerrit: https://gerrit.googlesource.com/gerrit/
+
+Tar: https://www.gnu.org/software/tar/
+
+Zoekt: https://github.com/google/zoekt
+
+Google Code Search: https://github.com/google/codesearch
+
+GerritForge: https://www.gerritforge.com/
+
+<!-- END SHOWNOTES -->
+
+<!-- START TRANSCRIPT -->
+
+**Beyang Liu:**
+
+Alright. Welcome back everyone to another edition of the Sourcegraph Podcast. Today, I'm here with Cassidy Williams. She's the director of Developer Experience at Netlify. She's also a prominent educator and presence in the React open source community who has helped many people get started in web development and coding in general.
+
+And prior to that, she held developer advocate and senior engineering positions at CodePen, Amazon, Clarify, Hacker Fund, and Venmo. And last but not least, she also has a great Twitter account, which I'm sure a lot of listeners are probably familiar with. At this point, it's very popular. She posts a lot of funny and on-point videos about developer life. Cassidy, welcome to the show.
+
+**Cassidy Williams:**
+
+Thank you so much for having me. And with the intro, I feel like such a big deal suddenly.
+
+**Beyang Liu:**
+
+You are a big deal. I think, what is it, you have like 170 thousand Twitter followers?
```

Comment:
```suggestion
You are a big deal. I think, what is it, you have like 170,000 Twitter followers?
```
Not sure what the house style says on this, but the numerals seem more helpful in this situation.